Onboarding responsive layout

jplorek-atix opened the pull request:
# Checklist

- [ ] JIRA - \<[LW-6739](https://input-output.atlassian.net/browse/LW-6739)>
- [ ] Proper tests implemented
- [ ] Screenshots added.

---

## Proposed solution

This PR adds responsiveness to the onboarding layout.


## Screenshots

https://github.com/input-output-hk/lace/assets/65184652/4d769dc1-ec10-482b-8e40-3d375556a065

DominikGuzei commented:
@jplorek-atix could we make the horizontal scrollbar in the same style as the vertical? (it's quite thick at the moment)

<img width="979" alt="grafik" src="https://github.com/input-output-hk/lace/assets/172414/075bd467-47f7-4ef7-acb5-381768623341">

jplorek-atix commented:
> @jplorek-atix could we make the horizontal scrollbar in the same style as the vertical? (it's quite thick at the moment)
> 
> <img alt="grafik" width="979" src="https://user-images.githubusercontent.com/172414/247643475-075bd467-47f7-4ef7-acb5-381768623341.png">

@DominikGuzei I noticed that when I was working on https://github.com/input-output-hk/lace/pull/59, so, I addressed this in there, I've updated the branch with main, now should look ok

<img width="938" alt="Screenshot 2023-06-21 at 15 18 01" src="https://github.com/input-output-hk/lace/assets/65184652/e92d9382-7a2f-4353-abfb-32028c0bf245">

DominikGuzei reviewed (approved):
LGTM 🚢

marcin-mazurek commented:
@jplorek-atix just checking if it's working as intended - if I narrow down the screen to 400px, the 3 column layout switches to one column. I understood that we should display the horizontal scrollbar but not change the layout?
![image](https://github.com/input-output-hk/lace/assets/6684554/a526d2a9-01ee-4cc5-aa62-74833b2379cb)

jplorek-atix commented:
> @jplorek-atix just checking if it's working as intended - if I narrow down the screen to 400px, the 3 column layout switches to one column. I understood that we should display the horizontal scrollbar but not change the layout?
> ![image](https://user-images.githubusercontent.com/6684554/248280198-a526d2a9-01ee-4cc5-aa62-74833b2379cb.png)

@marcin-mazurek It seems that there is a media query for 400px where changes the `flex-direction` from row to column, I'll address this and check if there are other places where this happens as well

jplorek-atix commented:
@allanleone There is this issue where the `Pin the wallet extension box` is to close to the the top and bottom, right now we are using a `min height` of 88px for the navigation and the items are centered without a explicit padding, what do you think would be the best approach to address this? or do you think is ok to leave it like this?

<img width="1427" alt="Screenshot 2023-06-23 at 11 39 15" src="https://github.com/input-output-hk/lace/assets/65184652/323c0cde-eac8-4b31-a2c3-c699c8151213">

allanleone commented:
> @allanleone There is this issue where the `Pin the wallet extension box` is to close to the the top and bottom, right now we are using a `min height` of 88px for the navigation and the items are centered without a explicit padding, what do you think would be the best approach to address this? or do you think is ok to leave it like this?
> 
> <img alt="Screenshot 2023-06-23 at 11 39 15" width="1427" src="https://user-images.githubusercontent.com/65184652/248301529-323c0cde-eac8-4b31-a2c3-c699c8151213.png">

For this "v1", I believe that "pin the wallet" popup can stay as it is, and, if required, to be placed as "absolute", to always stay at the top of menu/floating window...

 The important it's to the message be close enough of the extension button on chrome, pointing the user where to go for pinning the extension
 
 
![image](https://github.com/input-output-hk/lace/assets/8201486/02f995ed-0038-4f7b-8438-b09e2c009082)

marcin-mazurek commented:
@jplorek-atix Please check the issue with "Please read and accept the terms" tooltip positioning on the legal agreement screen:

https://github.com/input-output-hk/lace/assets/6684554/6d957fca-758d-4919-aeb0-3516dff9b62b

@jplorek-atix @allanleone question - would it not be easier to just set custom `min-width` on the body tag for onboarding only? It feels like it would let us avoid a lot of cases like this one and the previous one I mentioned

![Screenshot 2023-06-27 at 09 07 49](https://github.com/input-output-hk/lace/assets/6684554/b9a116ab-9d01-4da9-a71b-dc2afad1de09)

marcin-mazurek commented:
@bslabiak I can see some smoke test are failing, can you advise if we are ok to merge despite those failures?

bslabiak commented:
Yes we can proceed with merging, this issue(with tests) has been fixed on main branch.

DominikGuzei commented:
@marcin-mazurek @jplorek-atix looks good from my side - but what about your comment @marcin-mazurek about simplifying it by adding the min-width to body?

marcin-mazurek commented:
@DominikGuzei I understood that the `min-width` on the body level was actually an issue and taking it off and controlling on a lower level resolved the issue, but let's wait for @jplorek-atix can confirm that.

jplorek-atix commented:
> @DominikGuzei I understood that the `min-width` on the body level was actually an issue and taking it off and controlling on a lower level resolved the issue, but let's wait for @jplorek-atix to confirm that.

@DominikGuzei @marcin-mazurek yes, that is correct, I leave some examples with different min-width

980px

https://github.com/input-output-hk/lace/assets/65184652/f91a55ea-29e0-4750-a9df-155439dc5f5b

360px

https://github.com/input-output-hk/lace/assets/65184652/63e21093-c3d5-4fe9-91d0-53256d165654
